builtin/logical/pki/backend_test.go

```diff
@@ -3937,6 +3962,7 @@ func TestBackend_Root_FullCAChain(t *testing.T) {
 		{testName: "EC", keyType: "ec"},
 	}
 	for _, tc := range testCases {
+		tc := tc
```

Comment:
This is a bit curious, what are we trying to accomplish with this statement?

Reply:
That was here:

The [only gotcha in this approach](https://gist.github.com/posener/92a55c4cd441fc5e5e85f27bca008721) is that the call to t.Run(...) becomes
effectively async; this means we either need to not mark the test as
parallel or shadow any loop variables inside the scope of the loop to
allow the t.Run(...) call to have the correct copy.

Reply:
Cool thanks, and of course you added it in the description that I had not read *sorry*